### PR TITLE
Added functional tests for BindTo for Body & Form and changed BindingConfig for BindTo

### PR DESCRIPTION
**Pull request by jchannon**
Hopefully a fix for #1383

### CODE REVIEW

**Comment by khellang** [comment]
Moving this breaking change to the 2.0 milestone.

File: src/Nancy.Tests.Functional/Modules/SerializerTestModule.cs [hunk 1]
```diff
@@ -3,6 +3,8 @@
     using System;
     using System.Globalization;
```
Comment (khellang) [inline src/Nancy.Tests.Functional/Modules/SerializerTestModule.cs]: :fire:

File: src/Nancy.Tests.Functional/Tests/SerializerTests.cs [hunk 2]
```diff
@@ -31,8 +31,47 @@ public void Should_Serialize_To_ISO8601()
...
-            
-            Assert.Equal("{\"createdOn\":\"2013-12-25T12:10:30\"}", model);
+
+            Assert.Equal("{\"createdOn\":\"2013-12-25T12:10:30\",\"name\":null}", model);
+        }
+
+        [Fact]
+        public void Should_BindTo_Existing_Instance_Using_Body_Serializer()
+        {
+            //Given
```
Comment (khellang) [inline src/Nancy.Tests.Functional/Tests/SerializerTests.cs]: NIT; Would you mind going over these and add a space after `//`? :innocent:

File: src/Nancy.Tests.Functional/Tests/SerializerTests.cs [hunk 3]
```diff
@@ -31,8 +31,47 @@ public void Should_Serialize_To_ISO8601()
...
+
+            //Then
+            Assert.Equal("Marsellus Wallace", resultModel.Name);
+            Assert.Equal(new DateTime(2014, 01, 30), resultModel.CreatedOn);
+        }
+
+        [Fact]
+        public void Should_BindTo_Existing_Instance_Using_Form()
+        {
+            //Given
```
Comment (khellang) [inline src/Nancy.Tests.Functional/Tests/SerializerTests.cs]: :fire:

**Comment by khellang** [comment]
LGTM. Just a couple of nits and a rebase needed :shipit:

File: src/Nancy.Tests.Functional/Modules/SerializerTestModule.cs (hunk 1 above)
Comment (jchannon) [inline src/Nancy.Tests.Functional/Modules/SerializerTestModule.cs]: What?

File: src/Nancy.Tests.Functional/Tests/SerializerTests.cs (hunk 3 above)
Comment (jchannon) [inline src/Nancy.Tests.Functional/Tests/SerializerTests.cs]: huh?

**Pull request merged**